src/test/unit/algorithms/ConnectionsTest.cpp

```diff
@@ -227,16 +227,17 @@ namespace {
     // Add an additional synapse over the limit
     presynapticCell.idx = 52;
     synapse = connections.createSynapse(segment, presynapticCell, 0.52);
+    EXPECT_LT(synapse.idx, 2);
```

Comment:
If this fails, you want the test to continue on? And do you not know if the index is `0` or `1`? It would be better to assert an exact value if you can.